spec/actors/sufia/default_admin_set_actor_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'spec_helper'
+
+RSpec.describe Sufia::DefaultAdminSetActor do
+  let(:create_actor) do
+    double('create actor', create: true,
+                           curation_concern: work,
+                           update: true,
+                           user: depositor)
+  end
+  let(:actor) do
+    CurationConcerns::Actors::ActorStack.new(work, depositor, [described_class])
+  end
+  let(:depositor) { create(:user) }
+  let(:work) { build(:generic_work) }
+  let(:admin_set) { create(:admin_set) }
+  let(:permission_template) { create(:permission_template, admin_set_id: admin_set.id) }
+
+  describe "create" do
+    before do
+      allow(CurationConcerns::Actors::RootActor).to receive(:new).and_return(create_actor)
+    end
+
+    context "when admin_set_id is blank" do
+      let(:attributes) { { admin_set_id: '' } }
+      let(:default_id) { described_class::DEFAULT_ID }
+
+      it "creates the default AdminSet with a PermissionTemplate and calls the next actor with the default admin set id" do
```

Comment:
The variable name is `create_actor`, but the spec says `next actor`. Can you reconcile the language? I find the "next actor" to be a bit more intuitive (and it reconciles with the variable defined in the above class `Sufia:: DefaultAdminSetActor`).